src/StmtToHtml.cpp

```diff
@@ -347,36 +347,19 @@ class StmtToHtml : public IRVisitor {
     }
     void visit(const ProducerConsumer *op) {
         scope.push(op->name, unique_id());
-        stream << open_div("Produce");
+        stream << open_div(op->is_producer ? "Produce" : "Consumer");
         int produce_id = unique_id();
         stream << open_span("Matched");
         stream << open_expand_button(produce_id);
-        stream << keyword("produce") << " ";
+        stream << keyword(op->is_producer ? "produce" : "consume") << " ";
         stream << var(op->name);
         stream << close_expand_button() << " {";
         stream << close_span();;
-        stream << open_div("ProduceBody Indent", produce_id);
-        print(op->produce);
+        stream << open_div(op->is_producer ? "ProduceBody Indent" : "ConsumeBody Indent", produce_id);
```

Comment:
I was thinking it might need updating here: https://github.com/halide/Halide/blob/master/src/StmtToHtml.cpp#L650

But it looks like that's not the case.